CHANGELOG.md

```diff
@@ -8,6 +8,26 @@
 #### Fixed
 #### Removed
 
+## [v1.2.0](https://github.com/mesg-foundation/cli/releases/tag/v1.2.0)
+
+#### Added
+
+- ([#120](https://github.com/mesg-foundation/cli/pull/120)) Use latest MESG engine v0.12
+- ([#118](https://github.com/mesg-foundation/cli/pull/118)) Limit size of the upload during compilation to avoid mistakes (limit to 10MB)
```

Comment:
```suggestion
- ([#118](https://github.com/mesg-foundation/cli/pull/118)) Limit size of the upload during compilation to avoid mistakes (limit to 10MB).
```